test/integration-test/src/tests/smoke.rs

```diff
@@ -19,7 +19,7 @@ fn xdp() {
 #[integration_test]
 fn extension() {
     let (major, minor, _) = kernel_version().unwrap();
-    if major < 5 || minor < 9 {
+    if major < 5 || (minor == 5 && minor < 9) {
```

Comment:
shouldn't this be `major == 5 && minor < 9`?

Can't this whole check be `if (major, minor) < (5, 9) {`?